sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -63,20 +58,39 @@ add_sycl_plugin(unified_runtime
     "pi2ur.cpp"
     "ur/ur.hpp"
     "ur/ur.cpp"
-    "ur/adapters/level_zero/ur_level_zero.hpp"
-    "ur/adapters/level_zero/ur_level_zero.cpp"
     "ur/usm_allocator.hpp"
     "ur/usm_allocator.cpp"
     # These below belong to Unified Runtime PI Plugin only
     "pi_unified_runtime.hpp"
     "pi_unified_runtime.cpp"
     LIBRARIES
     Threads::Threads
+    UnifiedRuntimeLoader
+    UnifiedRuntime-Headers
+)
+
+# Build level zero adapter
+add_sycl_library("ur_adapter_level_zero" SHARED
```

Comment:
what will cause the UR loader to discover and load this adapter?

Reply:
There are couple additional steps to make this adapter loadable.  

1) Register adapter library filename with loader.  I opened PR here: https://github.com/oneapi-src/unified-runtime/pull/353/commits/53e7e7c1a4bc56e43411807602c7fc5cf4c6c18a
2) Add implementations of urGet*ProcAddrTable functions into adapter. 

Until both are completed sycl-ls will not work correctly.    I opened this PR as-is because there was some mention that this was blocking cuda adapter development. 

I will start working on adding (2) and validating with sycl-ls.

Reply:
My latest updated is fully loadable and sycl-ls works as expected

Reply:
confirmed on my side.